doc/examples/.promu.yml

```diff
@@ -3,6 +3,7 @@ verbose: false
 repository:
     path: github.com/prometheus/prometheus
 build:
+    cgo: false
```

Comment:
Do we have a go specific namespace? Might make more sense then adding that attribute to the general build namespace?